### ISSUE TITLE
Capture raw query details

### ISSUE THREAD

**Issue by jake-codes** [openzipkin/brave-cassandra#28]
It would be an excellent addition to have the raw query details captured in the trace. An example of this can be found in [Last Pickle's](https://github.com/thelastpickle/cassandra-zipkin-tracing) implementation.

**Comment by codefromthecrypt** [comment]
i think we do similar in mysql though obviously can be big.

client side, cassandra side or both?

**Comment by jake-codes** [comment]
Both would be ideal but either would still be helpful.

**Comment by codefromthecrypt** [comment]
@llinder @michaelsembwever I will presume you are ok with this heh. we can always do something different later also.

@jake-codes are you up to doing the pull request?

**Comment by llinder** [comment]
Makes sense to me and I think we actually do this internally for some services.  My only point of caution is that, while it's neat to see the query, it probably doesn't justify the extra storage costs for high traffic sites.  Unless we support some form of deduplication of tags, I don't know that there is a good solution for that besides adding some config to the instrumentation so folks can disable these extra tags.

**Comment by codefromthecrypt** [comment]
hmm we set the tag "cassandra.request" same as the link you mentioned.. https://github.com/thelastpickle/cassandra-zipkin-tracing/blob/master/src/main/java/com/thelastpickle/cassandra/tracing/ZipkinTracing.java#L146

**Comment by codefromthecrypt** [comment]
we also tag the query client side if it is a bound statement

```
    if (statement instanceof BoundStatement) {
      customizer.tag(
          CassandraTraceKeys.CASSANDRA_QUERY,
          ((BoundStatement) statement).preparedStatement().getQueryString());
    }
```

**Comment by codefromthecrypt** [comment]
I will be offline for a while, so if there's more follow-up I'd recommend whoever does to proceed with failing unit tests so that we aren't solving things already solved.
